Network december release --- DDoS policy customization

seyedfayaz opened the pull request:
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

jihochang committed:
Domain Services Swagger: Fix Put and Patch Domain Service Input

davidbargeron committed:
add Swagger spec for Bing Local Search API. (#4349)

* add Swagger spec for Bing Local Search API.

* update title "Local Search API" -> "Local Search Client" per yangyuan's comment on PR #4349. Also removed whitelist filter from SearchResponse filter.

* add readme.md

* fixed issues in readme.md and in Swagger spec caught by linter.

* suppress R3016 linter error wrt "_type" because it can't be changed by me.

* filtered ErrorResponse to "Errors" field only.

* add x-ms-examples

alexeldeib committed:
Add example with nested segmentation, fix number types, remove extraneous `produces` (#4311)

* add segmented metrics example, minor fixes

* add example, return metric get full to previous state

* remove duplicate file

* Fix examples

* fix go object type

* fix build issues on parameters

* fix go configuration

ananthradhakrishnan committed:
Changed the descriptions of multiple items to bring more clarity for users. (#4365)

* Some description text changes

* fix Go SDK codegen

wawon-msft committed:
Fixed non-existent extra parameter (bug) (#4364)

t-robalt committed:
[DO NOT MERGE] VA List Api version edit (#4257)

* VA List Api version edit

* Updated missing tests

* Update DatabaseVulnerabilityAssessmentListByDatabase.json

* Update DatabaseVulnerabilityAssessmentListByDatabase.json

* Update DatabaseVulnerabilityAssessmentListByDatabase.json

Commit:
Add Devtestlabs/2018-09-15 (#4357)

* Copy DTL 2016-05-15

* Add DTL 2018-09-15

* Update go and ruby readme

* Add package-2018-09 to java. Re-order go & ruby api tag

abhinav-ghai committed:
Add list and get software to GuestConfiguration (#4343)

* Add list and get software to GuestConfiguration

* Use newer API version for software

EvgenyAgafonchikov committed:
Move parameter to keep order in SDK (#4376)

shurd committed:
adding property ignoreMissingVNetServiceEndpoint to VirtualNetworkRule object in Cosmos DB (#4371)

mozehgir committed:
Update openapi-authoring-automated-guidelines.md (#4369)

michimune committed:
Fix network trace API bugs (Retake) (#4363)

* Fix network trace API bugs

* Fix network trace API V2

Commit:
Update swagger with DNSConfig and GPU (#4373)

solankisamir committed:
Move version 2018-01-01 to stable folder (#4384)

* Move version 2018-01-01 to stable folder

* remove stable folder

abhinav-ghai committed:
Revert Software List and GET from GuestConfig RP (private preview only) (#4380)

* Add list and get software to GuestConfiguration

* Use newer API version for software

* Revert software Swagger spec since these are private preview only

vchske committed:
Explicitly add MongoDbTasks.json to 2018-07-15 (#4358)

* Update MongoDbTasks.json

* Update readme.md

mozehgir committed:
Update openapi-authoring-automated-guidelines.md (#4381)

luluRagdoll committed:
Reservations RP 2018-06-01 Preview: Add new reserved resource enum type RedHat (#4370)

* Reservations RP 2018-06-01 Preview: Add new reserved resource type RedHat

* Update python version from 0.2.2 to 0.2.3 since this is a minor change

* Update the python version

* move Go codegen under preview directory

* Update readme.go.md

jhendrixMSFT committed:
fix Go SDK regex (#4388)

deepakabburi committed:
Added patch API to swagger doc in Visual studio RP (#4368)

* Added patch API to swagger doc for Visual Studio RP

* Fixed validation error

* Added properties to definition

* Fixed Account tag request

jaysterp committed:
ACR: Add contextAccessToken to Task step (#4389)

* update ACR swagger with contextAccessToken

* format json

jhendrixMSFT committed:
add datamigration tags to Go SDK build (#4391)

mozehgir committed:
Update PULL_REQUEST_TEMPLATE.md (#4392)

shakao committed:
[Visual Search API] Update encoding format documentation per PM request (#4383)

jhendrixMSFT committed:
enable swagger-to-sdk in visual search for Go SDK (#4398)

ilidemi committed:
Add facet sortBy and filter to ResourceGraph swagger (#4390)

sarangan12 committed:
Fix Version value to fix Ruby SDK Generation (#4400)

jianghaolu committed:
Add java multi api versions for network (#4402)

grlin committed:
Grlin/adding principal typeto ra (#4399)

* adding principal type to putRA

* updating version to 2018-09-01-preview

* base commit, version 2018-01-01-preview

* adding principal type + updating version number

* refactoring swagger to separate out API calls

* updating 2018-01-01-preview

* adding PrincipalType

* removing incorrect 2018-09-01 json file

* Fix package-2018-09-01-preview-only

yuzhangyi committed:
Enable delete activity (#4386)

* Enable delete activity

* Enable delete activity.

* Update

boxiao-o committed:
[DataFactory]Add sensitive fields extracted from connection string in swagger for AKV reference (#4375)

* Add sensitive fields extracted from connection string in swagger for akv reference

* Change sasUri type to object as it supports string, SecureString and AzureKeyVaultSecretReference, same with connection string.

jhendrixMSFT committed:
remove time from API version (#4382)

jianghaolu committed:
Add latest data migration service tag to java config (#4407)

* Add latest data migration service tag to java config

* Add azure-sdk-for-java entry

lubalibu committed:
Fix blob auditing documentation (#4403)

* FIx blob auditing swaggers documentation

* fix blob auditing alignment

yaakoviyun committed:
Update Vulnerability Assessment swaggers (#4394)

Update Vulnerability Assessment swaggers

v-djnisi committed:
Managed Instance short term retention (#3979)

* new changes

* First commit

* Readme.md change

seyedfayaz committed:
partial commit

Commit:
fixing sample names in cognitive services (#4422)

NMijat1024 committed:
Update swagger documetation to mark Collation property as writable. (#4314)

* Update swagger documetation to mark Collation property as writable.

Change Managed Instance collation property from read only to read and create.
Add collation to properties section in ManagedInstanceCreateMax example.

VSTS task: 12336257

* Remove 'cosmetics' from swagger.

* Remove comma as redundant

* Return to the initial, generated state.

Commit:
fixed some invalid values in dl account examples (#4414)

kpajdzik committed:
Add generating package.json to KeyVault data plane package (#4429)

kpajdzik committed:
Fix RM machinelearning TypeScript readme (#4427)

* Fix TypeScript machinelearning readme

* Refactor the readme file

amarzavery committed:
update folder to generate arm subscription to subscriptionManagement2 for node.js (#4431)

jhendrixMSFT committed:
remove duplicate entry for Go SDK build (#4411)

peetlotla committed:
Swagger closure for Azure Migration Hub Operations (#4330)

* Swagger coverage for Azure Migrate Hub operations

* Adding Operation_List.json to examples

* adding operation result definitions to migrate.json

* Update specification/migrateprojects/resource-manager/Microsoft.Migrate/preview/2018-09-01-preview/migrate.json

Fixing the JSON schema validation for migrate.json

* Update specification/migrateprojects/resource-manager/readme.md

Fixing the code generation path for preview swagger.

* Update specification/migrateprojects/resource-manager/Microsoft.Migrate/preview/2018-09-01-preview/migrate.json

Fixing potential SDK and security errors.

* Update specification/migrateprojects/resource-manager/Microsoft.Migrate/preview/2018-09-01-preview/migrate.json

Fixing potential new SDK errors.

* Proving x-ms-exmaples for response/request payloads for operations.

* Fixing schema validation for azure-sdk-for-go.

* Fixing schema validation for azure-sdk-for-go.

* Added missing patch and delete operations per the feedback.

* Swagger fixes for azure-sdk-for-go

* Swagger fixes for azure-sdk-for-go

* Swagger fixes for azure-sdk-for-go

* Fixing schema validation for azure-sdk-for-go.

* Adding delete method for migrate errors.

seyedfayaz committed:
tmp

lkems committed:
WebApps: Snapshots API for DR stamps (#4291)

* Include DRSecondary endpoint for GetPublishingProfileXml

* Snapshots API for DR stamps

yaakoviyun committed:
Adding ManagedInstanceVulnerabilityAssessments and ServerVulnerabilityAssessments (#4416)

lmazuel committed:
Split Python conf in it's own file (#4438)

lmazuel committed:
Py conf in its own file (#4439)

seyedfayaz committed:
ddosPolicyCustomization swagger swagger -- no examples

seyedfayaz committed:
swagger spec for pulic ip resource and ddos custom policy to support ddos policy customization

sumitabarahmand committed:
Add spec for alert repair/remediate (#4084)

* Add spec for alert repair/remediate

* Update remediate route to repair

* Add alert remediation properties to the alert model

* fix oav validation errors

* update remediationActionType items

* remove remediationActionType as that is not exposed on an alert

* change hasRemediationAction type to boolean

seyedfayaz committed:
examples added for ddos custom policy swagger spec

solankisamir committed:
Remove readonly attribute from identities collection in users (#4442)

EvgenyAgafonchikov committed:
Make NIC VM readOnly (#4444)

snhaphil committed:
Add Allowed Connections REST API (#4106)

* Added allowed connections to security.json

* Added allowed connections to security.json

* added examples

* modified the examples resources name

* Modified the asclocation in the examples file

* reformatted and added newline at the EOF

* modified examples description in security.json

* Fixing additionalProperties configuration to match type verification

* pr fixes

* modified the description to be aligned with ASC doc.

* inserted topologies addition to security.json

eliagrady committed:
Remove security contact phone field mandatory fields (#4417)

* Remove security contact phone field mandatory fields

* Update security contact examples

* Update security contact examples - refactored

* Update security contact examples - refactored - second iteration

jhendrixMSFT committed:
add guidance for example names (#4432)

Commit:
Azure Container Instance: Add start container group API (#4423)

* Update swagger with DNSConfig and GPU

* Azure Container Instance: Add start container group API

naveedaz committed:
Add GeoRegion property to DeletedSite model (#4448)

yangyuan committed:
[CognitiveServices] enable Bing Search/CustomSearch products to specify Endpoint (#4401)

jhendrixMSFT committed:
Move Ruby SDK config to its own config file (#4451)

The package-name variable for Ruby was colliding with the Go build.
Moved the Ruby config to its own file to avoid the collision.

mozehgir committed:
Update README.md (#4421)

seyedfayaz committed:
swagger spec for ddos policy customization

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-js

AutorestCI commented:
# Automation for azure-sdk-for-ruby

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-java

AutorestCI commented:
# Automation for azure-sdk-for-go

AutorestCI commented:
# Automation for azure-sdk-for-node

hbzhxying committed:
DataFactory: add table name in dataset (#4424)

number213 commented on specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/ddosCustomPolicy.json:
```diff
@@ -0,0 +1,291 @@
+{
```
Please put this file in `Microsoft.Network/stable/2018-10-01`, not `2018-08-01`

MikhailTryakhov replied:
Don't see that it's done...

seyedfayaz replied:
done now :)

number213 commented on specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/DdosCustomPolicyCreate.json:
```diff
@@ -0,0 +1,40 @@
+{
```
Please put this file in `Microsoft.Network/stable/2018-10-01`, not `2018-08-01` (same comment for other example jsons)

seyedfayaz replied:
done

MikhailTryakhov replied:
You should change folder where you put this file in the ```2018-10-01``` folder

seyedfayaz replied:
done

number213 commented on specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/publicIpAddress.json:
```diff
@@ -3,7 +3,7 @@
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2018-08-01"
+    "version": "2018-10-01"
```
Please add those changes to `Microsoft.Network/stable/2018-10-01/publicIpAddress.json`, not here

seyedfayaz replied:
done

danielqiu committed:
Adds storage account quick failover API (#4457)

promoisha committed:
Added missing PATCH operation to Api Issue resource (#4440)

VitaliyKurokhtin committed:
/identity endpoint added (#4460)

Commit:
fixing invalid location in examples (#4415)

mattbriggs committed:
Mdb 20181029 statepropdescriptions (#4452)

* Subscription Get state property desc

* Incorrect description for Deleted SubscriptionState

* resolving merge issue

seyedfayaz committed:
updating 2018-10-01 for ddos policy customization

tejaskarandikar commented on specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/ddosCustomPolicy.json:
```diff
@@ -0,0 +1,291 @@
+{
+  "swagger": "2.0",
```
Can you remove these file from the PR?

seyedfayaz replied:
done

tejaskarandikar commented on specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/ddosCustomPolicy.json:
```diff
@@ -0,0 +1,291 @@
+          "description": "The list of public IPs associated with the DDoS custom policy resource. This list is read-only."
+        },
+        "protocolCustomSettings": {
+          "readOnly": true,
```
Why is protocolCustomSettings readonly? We need to let users set this in the PUT?

seyedfayaz replied:
fixed

tejaskarandikar commented on specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/ddosCustomPolicy.json:
```diff
@@ -0,0 +1,291 @@
+    "ProtocolCustomSettingsFormat": {
+      "properties": {
+        "protocol": {
+          "readOnly": false,
```
Add enum for protocol please? Tcp, Udp and Syn

seyedfayaz replied:
done

tejaskarandikar commented on specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/publicIpAddress.json:
```diff
@@ -452,6 +456,37 @@
+          ],
+          "description": "The type of the DDoS protection plan associated with the public IP."
+        },
+        "domainNameLabel": {
```
Why is domainNameLabel, fqdn etc in the DDOS settings?

seyedfayaz replied:
removed

tejaskarandikar commented on specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/examples/DdosCustomPolicyCreate.json:
```diff
@@ -0,0 +1,40 @@
+    "ddosCustomPolicyName": "test-ddos-custom-policy",
+    "parameters": {
+      "location": "westus",
+      "properties": {}
```
You need to provide atleast 1 protocol custom setting in the custom policy else the PUT is not allowed

tejaskarandikar commented on specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/publicIpAddress.json:
```diff
@@ -358,6 +358,10 @@
           "description": "The FQDN of the DNS record associated with the public IP address."
         },
+        "ddosSettings": {
+          "$ref": "#/definitions/DdoSSettings",
```
This should probably be ./networkInterface.json#/definitions/DdoSSettings, we should validate

tejaskarandikar reviewed (changes requested):
Please take a look at the PR comments

MikhailTryakhov reviewed (changes requested):
please move all the files in ```2018-10-01``` folder. now they're in ```2018-08-01```

seyedfayaz committed:
removing changes made in 2018-08-01/

jianghaolu committed:
Add latest authorization tag to java config (#4406)

jhendrixMSFT committed:
Add maximum chunk size to data lake store docs. (#4464)

* Add maximum chunk size to data lake store docs.

* fix semantic validation errors

* deprecate operation FileSystem_Concat

seyedfayaz committed:
temp commit

seyedfayaz committed:
fixing ip resource references to ddos custom policies

zjpjack committed:
Supression rules for TagValue and TagDetails (#4469)

lmazuel committed:
Revert "Swagger closure for Azure Migration Hub Operations" (#4482)

* Revert "Add maximum chunk size to data lake store docs. (#4464)"

This reverts commit ab9c6ff77b4f15ee7880c8f860c7e96f8356e80d.

* Revert "Add latest authorization tag to java config (#4406)"

This reverts commit 7f0fd162676915ae5023e16854d9c118855b24ea.

* Revert "Mdb 20181029 statepropdescriptions (#4452)"

This reverts commit 90db24cbae7e52097ed21f595a169dd61800d070.

* Revert "fixing invalid location in examples (#4415)"

This reverts commit ffaf5955754300110f8419030baa6c77f93620c0.

* Revert "/identity endpoint added (#4460)"

This reverts commit 4f93deeb63f7b766f639ac2c697aba0260b62bbc.

* Revert "Added missing PATCH operation to Api Issue resource (#4440)"

This reverts commit 7df0ce599e8d6579acd109ed9f4805954304969f.

* Revert "Adds storage account quick failover API (#4457)"

This reverts commit 10ec74f40e6ae0fc0f735a21aa6697453674aea3.

* Revert "DataFactory: add table name in dataset (#4424)"

This reverts commit f8465dec826d1436bffe4bd061f4f15e58e91e0b.

* Revert "Update README.md (#4421)"

This reverts commit fa6a58438b2322f3e3fdacdea1cd54c612c39d09.

* Revert "Move Ruby SDK config to its own config file (#4451)"

This reverts commit 2ca50c83ad4d17f3248ddad9ca632d11b26ded53.

* Revert "[CognitiveServices] enable Bing Search/CustomSearch products to specify Endpoint (#4401)"

This reverts commit dc782b320161cb372dc3322aca860e452b76e9c5.

* Revert "Add GeoRegion property to DeletedSite model (#4448)"

This reverts commit a079367c50c15f74a2b801441e20b14bbc912bd8.

* Revert "  Azure Container Instance: Add start container group API (#4423)"

This reverts commit 2ea3e9ebf74c5bade9146643bfbef092dfe80118.

* Revert "add guidance for example names (#4432)"

This reverts commit 6780e8ad5b09fcc3dcbe9eec58671dc89295d3be.

* Revert "Remove security contact phone field mandatory fields (#4417)"

This reverts commit eaf464ee6a5803e1d1b0e16ede4e5af15efc0b25.

* Revert "Add Allowed Connections REST API (#4106)"

This reverts commit 2389da39084fb8880cde21111ff79bc72e74f830.

* Revert "Make NIC VM readOnly (#4444)"

This reverts commit 2d83450cbe91cc06d775ce384d8ab9c91589c894.

* Revert "Remove readonly attribute from identities collection in users (#4442)"

This reverts commit 7577df97b09389657606a5d1bc814ca7d44b1af9.

* Revert "Add spec for alert repair/remediate (#4084)"

This reverts commit 3f568b4898533a2699c90a7f451448dfebae140a.

* Revert "Py conf in its own file (#4439)"

This reverts commit 337376061dedcb5d6dfe7d2ddad59061c45f5710.

* Revert "Split Python conf in it's own file (#4438)"

This reverts commit 77cf607c0e92af7a2f6937aa5f30bc08673cb902.

* Revert "Adding ManagedInstanceVulnerabilityAssessments and ServerVulnerabilityAssessments (#4416)"

This reverts commit dec75495352902ebb6393d42c50465b6195f239d.

* Revert "WebApps: Snapshots API for DR stamps (#4291)"

This reverts commit 916ff2e6e2e722c8b6ca8c77149f6b2ab358332a.

* Revert "Swagger closure for Azure Migration Hub Operations (#4330)"

This reverts commit badd1de79620e5660c7e9a7b0495076152f66356.

danielqiu committed:
Adds Geo Replication Stats in Azure Storage Account's properties (#4455)

refortie committed:
Add 2 missing endpoints to newest versions Microsoft.Logic (#4332)

* Add missing endpoints to newest versions

* Changing copy error of POST to GET. Added missing page links

* Fix for top level properties errir

* Adding CloudErrors to new endpoints

* Rename CloudError to not a potentially reserved name and removed the x-ms-external flag

* Use already existing error response object

refortie committed:
Don't make ResourceReference id readonly, it's causing a bug in the SDK where an Logic Apps with integration accounts can't be saved (#4491)

idear1203 committed:
[HDInsight] - Support KV URL (#4449)

yareyes committed:
New Swagger specification for Microsoft.SqlVirtualMachine Resource Provider.  (#4385)

* Adding new SQL VM Resource Manager swagger and examples

* Final swagger

* Addressing code review comments, adding readme file

* Addressing code review comments

* Addressing code review comments

* Addressing comments

MikeDodaro committed:
Description of Autosuggest usage needs improvement. (#4490)

haitsongmsft committed:
add clientData properties to data migration tasks (#4462)

seyedfayaz committed:
minor fixes

ankushbindlish2 committed:
Microsoft.StorageSync : Update ApiVersion 2018-10-01 (#4249)

* Update ApiVersion , still in Preview folder. (Andre's change)

* fix Go SDK codegen for package-2018-10-01

* Update specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json

* Update Examples

* Update specification/storagesync/resource-manager/readme.typescript.md

* Fix example

* Update specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json

* Matching with service implementation

haitch committed:
split swagger to multiple area for easy upgrade (#4461)

* split swagger to multiple area for easy upgrade

* update suppression due to file split

sergey-shandar committed:
Update OAV (#4496)

xzsheng committed:
Add new resource provider Microsoft.EngagementFabric (#4480)

* Add new resource provider Microsoft.EngagementFabric

* Rename directory `2018-09-01-preview` to `2018-09-01`

* Fix incorrect folder name

* Add file read.typescript.md

* Add readme.go.md

* Fix incorrect package name

Commit:
Adding API For QnAMaker v4.0 (#4377)

* adding v4

* Addressing comments

* Use Endpoint Template

* Switch to parameter

* Formatting

* linter method issue

* examples ALTS

* example keys

* ops example

* Add kb example

* Adding Create Kb Example

* Download KB Example

* Fix Namespaces

* model validation errors

* change to octet stream

* add consume/produces

* remove response body

* fix set alt example

* Update Operation Ids

* Fix case of description

* Enums as strings

* Adding md files for other langs

* Update Ruby readme

* remove x-enumNames

jhendrixMSFT committed:
fix Go SDK codegen for engagementfabric (#4498)

mjfrb committed:
Master (#4450)

* fix readme.md to include 2016/06/01 with 2018/03/01 always

* add 2018-11-01-preview

* pr feedbac

* add cost center

* pr feedback

* pr feedback

nschonni committed:
typo: cdn/resource-manager/Microsoft.Cdn/cdn.json (#4516)

- Supplemnetal -> Supplemental
- pictires -> pictures
- exisitng -> existing
- Double word "the"
- acess -> access
- adress -> address
- reponse -> response

jhendrixMSFT committed:
add azure-kusto to Go SDK (#4523)

nschonni committed:
typo: cosmos-db/resource-manager/Microsoft.DocumentDB/cosmos-db.json (#4513)

- defintions -> definitions
- parition -> partition

grlin committed:
Grlin/adding principal typeto ra (#4525)

* adding principal type to putRA

* updating version to 2018-09-01-preview

* base commit, version 2018-01-01-preview

* adding principal type + updating version number

* refactoring swagger to separate out API calls

* adding PrincipalType

* removing incorrect 2018-09-01 json file

* adding PrincipalType to RoleAssignmentPropertieswithScope

* refactoring Go out of readme

hvermis committed:
[Datafactory]Adding get Dataplane access API (#3957)

* Adding get Dataplane readonly token API

* Updated the casing and the example

* Updating dataplane access APIs

* Updating examples

zjpjack committed:
Change the suppress type name (#4499)

nschonni committed:
typo: Fix apimanagement/resource-manager/Microsoft.ApiManagement/apimbackends.json (#4510)

- parition -> partition
- Reconect -> Reconnect

nschonni committed:
typo: Fix Microsoft.Web/CommonDefinitions.json (#4504)

- Double word "to"
- resourse -> resource
- mimuntes -> minutes
- specificed -> specified
- successfull -> successful
- crendentials -> credentials

nschonni committed:
typo: iothub/resource-manager/Microsoft.Devices/iothub.json (#4512)

- Double word "the"
- parition -> partition

saravanan211094-git committed:
DataBox : Swagger and example changes for next version of client DLL (#4393)

* Managed disk updates

* changes

* changing examples for managed disk details

* changing readme and dll version in examples

danielqiu committed:
Adds canFailover boolean flag under storage account's GeoReplicationStats properties (#4532)

SamhitaK6 committed:
[Microsoft.Billing]: Updating swagger with rename changes required. (#4495)

* Microsoft.Billing: Gtm ImportRequest swagger

* Taking CR comments.

* Taking CR comments.

* [Microsoft.Billing]: Updating swagger with rename changes required.

nschonni committed:
typo: web/resource-manager/Microsoft.Web/AppServiceEnvironments (#4527)

- enviroment -> environment

nschonni committed:
typo: Fix customer-insights/resource-manager/Microsoft.CustomerInsights/customer-insights (#4517)

- interation -> interaction
- retreive -> retrieve
- exisiting -> existing
- refered -> referred
- assiciated -> associated
- monified -> modified
- mangement -> management
- ingored -> ignored
- c onnector -> connector
- singnas -> singnals

lmazuel committed:
Change default generation to TextAnalytics 2.1 (#4533)

leonardbf committed:
NFSAAS-1505 minor update and validation fixes (#4520)

nschonni committed:
typo: policyinsights/resource-manager/Microsoft.PolicyInsights/policyEvents (#4535)

- seperated -> separated

nschonni committed:
typo: azsadmin/resource-manager/storage/Microsoft.Storage.Admin/acquisitions (#4544)

- acquistions -> acquisitions
- Double word "the"

nschonni committed:
typo: batchai/resource-manager/Microsoft.BatchAI/BatchAI (#4546)

- abouth -> about
- everytime -> every time
- choosen -> chosen
- succesfully -> successfully
- downloand -> download
- parmeter -> parameter
- respnose -> response
- instumentation -> instrumentation
- concatinate -> concatenate
- unmouted -> unmounted
- mouted -> mounted

nschonni committed:
typo: datalake-store/data-plane/Microsoft.DataLakeStore/filesystem (#4537)

- seperated -> separated
- lexographical -> lexicographical
- contet -> content

nschonni committed:
typo: servicefabric/data-plane/Microsoft.ServiceFabric/stable/6.2/servicefabric (#4528)

- specificed -> specified
- sucessful -> successful
- resturns -> returns
- commited -> committed
- Double word "maximum"
- Double word "with"

nschonni committed:
typo: servicefabric/data-plane/Microsoft.ServiceFabric/CreateService-4 (#4514)

- Parition -> Partition

nschonni committed:
typo: apimanagement/control-plane/Microsoft.ApiManagement/apimbackends.json (#4511)

- parition -> partition

bashahee committed:
Update sensitivityLabels.json (#4479)

* Add sensitivityLabels.json to readme.md

Removing duplication of sensitivityLabels.json files, and updating readme.md

* Update sensitivityLabels.json

* Update readme.md

* Update readme.md

nschonni committed:
typo: applicationinsights/resource-manager/Microsoft.Insights/componentFeaturesAndPricing_API (#4552)

- capabilites -> capabilities
- volumne -> volume
- applciation -> application
- Insigths -> Insights
- Double word "meter"
- Desplay -> Display
- vaule -> value

nschonni committed:
typo: storage/resource-manager/Microsoft.Storage/blob (#4551)

- extented -> extended

nschonni committed:
typo: compute/quickstart-templates/swagger (#4557)

- deploymentoutput -> deployment output
- paramaters -> parameters

nschonni committed:
typo: storage/resource-manager/Microsoft.Storage/storage (#4549)

- possiblly -> possibly
- availabity -> availability
- vaules -> values
- speicific -> specific
- acount -> account
- fecthing -> fetching
- sku -> SKU
- acls -> ACLs

nschonni committed:
typo: mediaservices/resource-manager/Microsoft.Media/Encoding (#4542)

- Double word "will"
- Double word "of"
- audi -> audio
- extraced -> extracted

nschonni committed:
typo: datafactory/resource-manager/Microsoft.DataFactory/entityTypes/LinkedService (#4536)

- retrive -> retrieve
- seperate -> separate
- Serivce- > Service

nschonni committed:
typo: eventhub/resource-manager/Microsoft.EventHub/EventHub (#4553)

- successfuly -> successfully
- vaule -> value
- Autorization -> Authorization
- Regenrate -> Regenerate
- Messagning -> Messaging
- envokes -> Invokes
- neeeds -> needs
- achive -> archive
- anme -> name
- Usermetadata -> User Metadata
- reponse -> response
- pairning -> pairing

nschonni committed:
typo: seperate -> separate (#4539)

zjpjack committed:
Add type for DeploymentExtend and ResourceGroup (#4501)

nschonni committed:
typo: mediaservices/resource-manager/Microsoft.Media/StreamingPoliciesAndStreamingLocators (#4543)

- proprty -> property
- Stremaing -> Streaming
- sepearate -> separate
- Acquistion -> Acquisition
- speicified -> specified
- Configuations -> Configuration
- Doulbe word "of"
- Doulbe word "the"

eliagrady committed:
rename folder paths for autorest code generation (#4562)

nschonni committed:
typo: applicationinsights data-lane and resource-manager (#4554)

- metatadata -> metadata
- assmebly -> assembly
- doens't -> doesn't
- reponse -> response
- createion -> creation
- Applciation -> Application
- aditional -> additional
- an workitem -> a work item
- detaile d -> detailed
- defintions -> definitions
- definitioned -> definitions

nschonni committed:
typo: alertsmanagement/resource-manager/Microsoft.AlertsManagement/AlertsManagement.json (#4515)

- Succesfully -> Successfully
- Defaut -> Default
- seperated -> separated

nschonni committed:
typo: README.md (#4541)

- Doulbe word "of"
- MSFT -> Microsoft

Commit:
Fix endpoint name clash (#4548)

seyedfayaz committed:
Merge branch 'network-november-release' of https://github.com/Azure/azure-rest-api-specs into network-november-release

nschonni committed:
typo: profile (#4555)

- avaialbility -> availability
- Remove extra tailing comma
- Trim whitespace
- resource -> resource

nschonni committed:
typo: CONTRIBUTING.md (#4540)

- Double word "of"
- sytactically -> syntactically
- properities -> properties

nschonni committed:
typo: Documentation folder (#4556)

- Add missing code fence languages
- userid -> userId
- defintion -> definition
- definining -> defining
- indcate -> indicate
- exension -> extension
- paramemter -> parameter
- acronmys -> acronyms
- referenes -> references
- recommanded -> recommended
- refere -> refer
- exaclty -> exactly
- erraneous -> erroneous
- shema -> schema
- flexibily -> flexibility
- reponse -> response
- atleast -> at least
- availabilty -> availability
- relvant -> relevant
- Needlees -> Needless
- throughly -> thoroughly
- continously -> continuously
- specfication -> specification
- valueable -> valuable

QingqingYuan2 committed:
Deprecate old version of RDBMS (#4526)

* Deprecate pre-GA APIs

* update readme files

seyedfayaz committed:
fixing ddossetting definition in public ip address json

seyedfayaz committed:
1) misc fixes, 2) added pulic ip address creation with ddos custom policy

MikhailTryakhov commented:
@seyedfayaz there is no activity resolving comments provided, milestone is missed. please target on the next monthly branch (review milestones section in one note)

MikhailTryakhov committed:
Network november release (#4509)

* Adds base for updating Microsoft.Network from version stable/2018-08-01 to version 2018-10-01

* Updates readme

* Updates API version in new specs and examples

* [Networkwatcer] add FlowLogFormatParameters (#4374)

* Update specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/networkWatcher.json

[Networkwatcher] add a new field "Format" with new type"FlowLogFormat" to FlowLogProperties

* Revert "Update specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/networkWatcher.json"

This reverts commit f4f438af5a3d42a21936ecb6d3f661f3fa4a4442.

* Update specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json

[networkwatcer] add FlowLogFormatParameters

* [Network wathcer] add a new field trafficAnalyticsInterval for TA (#4420)

* Make NIC VM ref readonly (#4443)

* Mark private IP address read-only (#4447)

* change container nics property type on container nic config to resourceid (#4459)

* Port fix from PR 4459 to 2018-10-01 (#4463)

* Application Gateway - Adds Rewrite rule set for Header CRUD (#4331)

* Adds base for updating Microsoft.Network from version stable/2018-08-01 to version 2018-10-01

* Updates readme

* Updates API version in new specs and examples

* Adds Rewrite rule set for Header CRUD

Adds the structure of rewriteRuleSets in the applicationGatewayProperties.
Adds recursively the sub structures in the rewriteRuleSet.
Adds the reference to rewriteRuleSet in the requestRoutingRule for both Basic rule and Path based rule.

* Adds examples for the Header CRUD in application gateway

* Fix the examples; replacing rules with rewriteRules.

* Fixing the properties of application Gateway rewriteRule

* Revert "Fixing the properties of application Gateway rewriteRule"

This reverts commit 05c9c9103af1da0162b6cc0a04c4b37b4360f66c.

* Fixing the properties of application Gateway rewriteRule

* Mark the provisioning state as readOnly

* Mark etag as readOnly

* Revert "Model ContainerNic refs under ContainerNicConfig as sub resources" (#4467)

* Revert "Application Gateway - Adds Rewrite rule set for Header CRUD (#4331)"

This reverts commit 633d12d5b552b102f874182b8f50266eb97fdc85.

* Revert "Port fix from PR 4459 to 2018-10-01 (#4463)"

This reverts commit 5ea0c7b8bcf16d992bb1f2b5f3b52208354a4391.

* Revert "change container nics property type on container nic config to resourceid (#4459)"

This reverts commit baf31d901254dd4bf4f1813e9faeec1c9805f775.

* Fixes the missing array for the header actions (#4497)

* Network py 2018-10

* add package-2018-10 to Go SDK codegen

* Application gateway Identity and Keyvault support (#4387)

* identity and keyvault

* Capitalization comment

* Ported fix from master branch (#4547)

seyedfayaz committed:
fixed protocol in ddos custom policy creation example

Commit:
Change path for generated sdk (#4585)

mhofmann-hb committed:
Update aliases.json (#4586)

Update alias.json with an Alias for Windows Server 2019 Datacenter

jhendrixMSFT committed:
Fix type-o in adhybridhealthservice model (#4587)

Move Go SDK config to its own config file

jhendrixMSFT committed:
Add Go SDK build for sqlvirtualmachine (#4588)

nschonni committed:
typo: datafactory/resource-manager/Microsoft.DataF (#4573)

actory

- availably -> availability
- Serivce -> Service
- specificed -> specified
- occurence -> occurrence
- everytime -> every time
- Double word "the"
- Rrepository -> Repository
- proprety -> property

yugangw-msft committed:
add customKeyIdentifier to password credential (#4507)

* add customKeyIdentifier to password credential

* add creds in app object

* custom id(thumbprint) for cert creds should be string type

ankushbindlish2 committed:
Microsoft.StorageSync - Fix Issue 4559 and other known problems with preview version (#4565)

* Update storagesync.json

* Update storagesync.json

* Fix Examples

nschonni committed:
typo: analysisservices/resource-manager/Microsoft.AnalysisServices (#4583)

- Supends -> Suspends
- enableing -> enabling
- availibility -> availability

dedhar committed:
Add support for List Operation on global reach connections (#4589)

* Adds base for updating Microsoft.Network from version stable/2018-08-01 to version 2018-10-01

* Updates readme

* Updates API version in new specs and examples

* [Networkwatcer] add FlowLogFormatParameters (#4374)

* Update specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/networkWatcher.json

[Networkwatcher] add a new field "Format" with new type"FlowLogFormat" to FlowLogProperties

* Revert "Update specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/networkWatcher.json"

This reverts commit f4f438af5a3d42a21936ecb6d3f661f3fa4a4442.

* Update specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/networkWatcher.json

[networkwatcer] add FlowLogFormatParameters

* [Network wathcer] add a new field trafficAnalyticsInterval for TA (#4420)

* Make NIC VM ref readonly (#4443)

* Mark private IP address read-only (#4447)

* change container nics property type on container nic config to resourceid (#4459)

* Port fix from PR 4459 to 2018-10-01 (#4463)

* Application Gateway - Adds Rewrite rule set for Header CRUD (#4331)

* Adds base for updating Microsoft.Network from version stable/2018-08-01 to version 2018-10-01

* Updates readme

* Updates API version in new specs and examples

* Adds Rewrite rule set for Header CRUD

Adds the structure of rewriteRuleSets in the applicationGatewayProperties.
Adds recursively the sub structures in the rewriteRuleSet.
Adds the reference to rewriteRuleSet in the requestRoutingRule for both Basic rule and Path based rule.

* Adds examples for the Header CRUD in application gateway

* Fix the examples; replacing rules with rewriteRules.

* Fixing the properties of application Gateway rewriteRule

* Revert "Fixing the properties of application Gateway rewriteRule"

This reverts commit 05c9c9103af1da0162b6cc0a04c4b37b4360f66c.

* Fixing the properties of application Gateway rewriteRule

* Mark the provisioning state as readOnly

* Mark etag as readOnly

* Revert "Model ContainerNic refs under ContainerNicConfig as sub resources" (#4467)

* Revert "Application Gateway - Adds Rewrite rule set for Header CRUD (#4331)"

This reverts commit 633d12d5b552b102f874182b8f50266eb97fdc85.

* Revert "Port fix from PR 4459 to 2018-10-01 (#4463)"

This reverts commit 5ea0c7b8bcf16d992bb1f2b5f3b52208354a4391.

* Revert "change container nics property type on container nic config to resourceid (#4459)"

This reverts commit baf31d901254dd4bf4f1813e9faeec1c9805f775.

* Fixes the missing array for the header actions (#4497)

* Network py 2018-10

* add package-2018-10 to Go SDK codegen

* Application gateway Identity and Keyvault support (#4387)

* identity and keyvault

* Capitalization comment

* Ported fix from master branch (#4547)

* Add support for list api for global reach connections

* address farhan comment

nschonni committed:
typo: addons/resource-manager/Microsoft.Addons (#4580)

- camonical -> canonical

nschonni committed:
typo: devtestlabs/resource-manager/Microsoft.DevTestLab/DTL (#4538)

- occured -> occurred
- Reponse -> Response
- seperated -> separated

vivek-microsoft committed:
Updating PUT notificationSettings api (#4445)

* Moving to Public Repository

Moving to Public Repository

* moving inside preview folder

* updating relative path

* Updating readme with code generation instructions

Updating readme with code generation instructions

* Updating tag

Updating tag

* adding PUT notificationSettings api

* adding example file

adding example file for notification settings PUT api

* Fix example format

* fix validation error

* fix validation error

* validation error fix

* fix validation error

* fix example

* fix example

* fix example

* fix example

* Removing python sdk instruction from readme.md

* Review comments

* Deleting temporary file.

* Adding examples

* review comments

* Updating response format

* Fixing json format

* Restoring Python sdk in readme.md

* Updating id in response

* manual edit to refer to default parameter

Commit:
Add readme.nodejs.md and readme.typescript.md for azure-kusto (#4592)

veronicagg committed:
Adds base for updating Microsoft.Network from version stable/2018-10-01 to version 2018-12-01

veronicagg committed:
Updates readme

veronicagg committed:
Updates API version in new specs and examples

seyedfayaz committed:
test

seyedfayaz committed:
test

seyedfayaz committed:
updating api version

seyedfayaz committed:
ddos custom policy swagger -- december branch

seyedfayaz committed:
minor update

seyedfayaz committed:
minor fix

seyedfayaz commented:
creating a new pr
